Frameworks/UIKit/UIViewController.mm

```diff
@@ -491,9 +491,15 @@ - (CGRect)_modalPresentationFormSheetFrame {
 
 - (BOOL)_hidesParent {
     UIModalPresentationStyle style = [self modalPresentationStyle];
-    if ((style == UIModalPresentationFormSheet || style == UIModalPresentationPopover) && DisplayProperties::IsTablet()) {
+
+    if (style == UIModalPresentationPopover) {
+        // Popovers (which don't hide their parent) are always shown regardless of tablet mode.
```

Comment:
To clarify, specifying presentationStyle = UIModalPresentationPopover under a non-tablet UIInterfaceIdiom resulted in presentation of a full screen modal previously.